### PR TITLE
3638 update openstack page

### LINKED ISSUE #3638: Beta: update bottom 1/3rd of /openstack page
the [copy doc](https://docs.google.com/document/d/1eqHK0hc9v7Mn-XV36QRyNkJenTXgBQR_-RULKm5rZ5o/edit#) has different resources than the live 'Learn more about OpenStack' section and a everything else below...

Also 'Openstack' should be spelled 'OpenStack'

### PR DESCRIPTION
**Pull request by steverydz**
## Done

Update openstack page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: <http://0.0.0.0:8001/openstack>
- See that the page matches the [copy doc](https://docs.google.com/document/d/1eqHK0hc9v7Mn-XV36QRyNkJenTXgBQR_-RULKm5rZ5o/edit#)

## Issue / Card

Fixes #3638

### CODE REVIEW

File: templates/openstack/index.html [hunk 1]
```diff
@@ -217,6 +217,60 @@ <h2>The number one cloud and container operating system, by far</h2>
   {% include "shared/_resources_openstack.html"%}
 </section>
 
-{% include "shared/contextual_footers/_contextual_footer.html"  with first_item="_fully_managed_openstack_cloud" second_item="_consulting_packages" third_item="_install_openstack_now" %}
+<section class="p-strip is-bordered">
+  <div class="row p-divider">
+    <div class="col-4 p-divider__block">
+      <h3 class="p-heading--four">Fully managed private OpenStack cloud</h3>
+      <p>Start your private cloud journey with a low-risk managed OpenStack by Canonical. We help you design, deliver and operate your cloud, enabling you to focus on your apps while proving the business case for OpenStack.</p>
+      <p><a href="/openstack/managed">Learn about BootStackLearn about BootStack&nbsp;&rsaquo;</a></p>
```
Comment (anthonydillon) [inline templates/openstack/index.html]: Doubling

**Comment by frankiegranato** [comment]
- [ ] Wong pricing table at top of page (I think Peter has fixed this in another PR)
- [ ] Matrix grid should be used 
openstack-index-matrix
openstack-index-wrong-pricing

**Comment by frankiegranato** [comment]
All good - apart from - I just noticed that the page description doesn't match the copydoc.

**Comment by steverydz** [comment]
@frankiegranato Fixed

**Pull request merged**